api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,4 @@
 //go:build !ignore_autogenerated
-// +build !ignore_autogenerated
```

Comment:
Hmm, not sure about this one, this still give this diff on go1.21 locally. This seems to be more related `controller-gen` v0.15.0 instead requiring go1.22. So people's `authorino/bin/controller-gen` might be outdated if it's already been downloaded before since I believe it won't be redownloaded by the makefile command if it's already present 🤔 

<img width="720" alt="image" src="https://github.com/user-attachments/assets/989c2b43-1071-4d56-bca0-a95644b141d5">